src/exercises/day-1/book-library.rs

```diff
@@ -100,7 +100,7 @@ impl Library {
 // implement the missing methods. You will need to update the
 // method signatures, including the "self" parameter!
 fn main() {
-    let library = Library::new();
+    let mut library = Library::new();
```

Comment:
Hi @sahennenkamp, you're completely correct that you need to add `mut` here.

However... :smile: That is something the students should discover for themselves when solving the exercise.

Can you instead update the comments above to let people know that they (might) have to make things mutable?

Reply:
Hey @mgeisler! Thanks for the input, that makes lots of sense! I'll open a new one that edits the comments. I was thinking about also editing the solution to include the `mut` if that works with you?

Reply:
oh nvm, I see that this is the soln: https://github.com/google/comprehensive-rust/blob/c212a473ba4d060b209b654c0dd403328d6e137f/src/exercises/day-1/solutions-afternoon.md?plain=1#L8
I'll just update the exercise comments